client/gutenberg/extensions/presets/jetpack/editor-shared/jetpack-plugin-sidebar.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import { createSlotFill } from '@wordpress/components';
+import { Fragment } from '@wordpress/element';
+import { PluginSidebar, PluginSidebarMoreMenuItem } from '@wordpress/edit-post';
+import { registerPlugin } from '@wordpress/plugins';
+
+/**
+ * Internal dependencies
+ */
+import './jetpack-plugin-sidebar.scss';
+import JetpackLogo from 'components/jetpack-logo';
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+
+const { Fill, Slot } = createSlotFill( 'JetpackPluginSidebar' );
+
+const JetpackIcon = () => <JetpackLogo />;
```

Comment:
I don't quite get why we need this indirection? Why not use `<JetpackLogo />` inline where this PR is now using `<JetpackIcon />`?

Reply:
Good call - that's a remnant from something else I had there. Removed in e02e6248aa11979164f4ef0e8a3883171836b9fe.